<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roi_image</class>
 <widget class="QDialog" name="roi_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="roi1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>501</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #222;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="roi3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>501</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #222;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="roi2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>501</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #222;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="roi4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>410</y>
     <width>501</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #222;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelv1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #222;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#eeeeee;&quot;&gt;V1: &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#eeeeee;&quot;&gt;Quano Ave. (Cebu bound)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>501</width>
     <height>4</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelv1_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #222;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#eeeeee;&quot;&gt;V3:&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#eeeeee;&quot;&gt; E.O. Perez (CDU Bound)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>501</width>
     <height>4</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelv1_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #222;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#eeeeee;&quot;&gt;V2: &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#eeeeee;&quot;&gt;Quano Ave. (Parkmall Bound)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_7">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>501</width>
     <height>4</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelv1_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #222;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#eeeeee;&quot;&gt;V4:&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#eeeeee;&quot;&gt; E. O. Perez (Terminal Bound)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_8">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>501</width>
     <height>4</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
